src/prog_translator.h

```diff
@@ -27,14 +36,7 @@ class ExchangeNodeGroup;
 class ProgTranslator {
  public:
   /// @brief struct to hold all problem instance state
-  struct Context {
-    std::vector<double> obj_coeffs;
-    std::vector<double> row_ubs;
-    std::vector<double> row_lbs;
-    std::vector<double> col_ubs;
-    std::vector<double> col_lbs;
-    CoinPackedMatrix m;
-  };
+  typedef ProgTranslatorContext Context;
```

Comment:
Note that this typdef, with the rest of this changeset is not used anywhere else in the code base.